### ISSUE TITLE
display AFCH by default

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia-gadgets/afch-rewrite#263]
Currently, when a user enables the AFCH gadget for the first time, it does not display until you visit a draft, then click More -> Review (AFCH Beta)

This is unintuitive, and caused @MPGuy2824 to have some problems when activating the gadget for the first time.

I propose it, at a minimum, be enabled by default.

Additionally, maybe we should remove the option to hide it. Is there a situation where someone would want to have the gadget enabled but not display AFCH? They can just go to Special:Preferences and turn the gadget off instead.

Thoughts?

**Comment by primefac** [comment]
Um.... AFCH doesn't show up on non-draft pages, so why would it display elsewhere? I think I'm missing the "bug" here.

**Comment by NovemLinguae** [comment]
Sorry, I explained poorly. Let me try again.

So if you've never used AFCH before, and you turn on the gadget, then visit a draft, it doesn't show up by default. You have to click "More -> Review (AFCH Beta)"

I propose that we eliminate the "More -> Review (AFCH Beta)" step by making it display by default.

**Comment by primefac** [comment]
Isn't that the normal way it works? I have gone More > Review since I started reviewing...

**Comment by NovemLinguae** [comment]
Oh, I have the AFCH preference "Automatically open the review panel on AfC submissions" turned on. I guess I got used to that and thought it was the default.

**Comment by primefac** [comment]
I think this is a question the Project should answer... posted for comment [here](https://en.wikipedia.org/wiki/Wikipedia_talk:WikiProject_Articles_for_creation#AFCH_-_display_by_default%3F).

**Comment by KylieTastic** [comment]
"Is there a situation where someone would want to have the gadget enabled but not display AFCH?" - Er YES! If I'm working on a draft rather than reviewing it I don't want that huge extra banner at the top.

**Comment by NovemLinguae** [comment]
No consensus in the linked discussion to implement this. Closing.
